server/controllers/votesController.js

```diff
@@ -0,0 +1,82 @@
+import 'babel-core/register';
+import 'babel-polyfill';
+import db from '../models/index';
+
+
+/**
+ *
+ *
+ * @export
+ * @class Vote
+ */
+export default class Vote {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} with a successful message and recipe with the vote
+   * @memberof Vote
+   */
+  static async voteRecipe(req, res) {
```

Comment:
### Critical - consistency check
If you are implementing `async/await`, it has to be consistent across your classes. Please ensure this is done.

Reply:
Fixed.